src/service/standard-actions-impl.js

```diff
@@ -68,6 +68,17 @@ export class StandardActions {
     /** @const @private {!./viewport/viewport-impl.Viewport} */
     this.viewport_ = Services.viewportForDoc(ampdoc);
 
+    // A meta[name="amp-action-whitelist"] tag, if present, contains, 
+    // in its content attribute, a whitelist of actions on the special AMP target. 
+    const meta =
+        this.ampdoc.getRootNode().head.querySelector('meta[name="amp-action-whitelist"]');
```

Comment:
80 column limit. `gulp presubmit` should catch this.

Reply:
done this for the actual pull request #13192